Resources/Prototypes/Entities/Objects/Consumable/Food/Baked/cake.yml

```diff
@@ -782,10 +800,12 @@
   - type: SolutionContainerManager
     solutions:
       food:
-        maxVol: 48
+        maxVol: 125 # SUPER dense cake
         reagents:
         - ReagentId: Nutriment
-          Quantity: 48
+          Quantity: 80
+        - ReagentId: Sugar
+          Quantity: 30
```

Comment:
yeah that works out with similar scaling to the mundane cakes & incorporating the source 30u of sugar